pkg/service/replay/replay.go

```diff
@@ -89,21 +89,30 @@ func (r *Replayer) Start(ctx context.Context) error {
 			utils.LogError(r.logger, err, "failed to stop recording")
 		}
 	}()
+	if _, err := os.Stat(r.config.Path); os.IsNotExist(err) {
```

Comment:
Please add this validation in validateflags https://github.com/keploy/keploy/blob/cca2d422993deb6180a45d96dedf7fc3dba2e6a0/cli/provider/cmd.go#L260